### ISSUE TITLE
remove admin/default from admin set membership service

### ISSUE THREAD

**Issue by lsat12357** [OregonDigital/hyrax-migrator#138]
we don't actually want to use admin/default, and it's a hassle to change the admin set id, so before we start migrating, do we want to change the fallback behavior and have the service fail rather than assign admin/default to a work?

**Comment by lsat12357** [comment]
Discussed at metadata/migration mtg on 20200116, answer is yes, fail the admin_set_service if a set can't be assigned either by primary set or institution.

**Comment by lsat12357** [comment]
duplicate of #222
